pkgs/applications/networking/cluster/spacegun/default.nix

```diff
@@ -1,16 +1,21 @@
-{ pkgs, nodejs, stdenv, lib, ... }:
+{ pkgs, fetchFromGitHub, nodejs, stdenv, lib, ... }:
 
 let
-
-  packageName = with lib; concatStrings (map (entry: (concatStrings (mapAttrsToList (key: value: "${key}-${value}") entry))) (importJSON ./package.json));
+  src = fetchFromGitHub {
+    owner = "dvallin";
+    repo = "spacegun";
+    rev = "v0.3.3";
```

Comment:
Does this format not contain pname + version or name + version?

Reply:
this is not a version of derivation but fetchFromGithub argument. Are you sure this comment is relevant here?